apps/web/src/components/ParticipantWindow/ParticipantWindow.tsx

```diff
@@ -56,14 +54,7 @@ export default function ParticipantWindow() {
     <aside className={clsx(classes.participantWindowContainer, { [classes.hide]: !appState.isParticipantWindowOpen })}>
       <ParticipantWindowHeader />
```

Comment:
Something funny is happening with this header when the list gets really long. The header shrinks some:
![image](https://user-images.githubusercontent.com/12685223/148449735-f614daca-2c69-4362-bab0-4e3278943aa6.png)
Seems to be caused by `display: flex`. An easy fix might be `flex-shrink: 0`.